Source/WarpX.cpp

```diff
@@ -566,7 +585,7 @@ WarpX::ReadParameters ()
         particle_pusher_algo = GetAlgorithmInteger(pp, "particle_pusher");
         maxwell_fdtd_solver_id = GetAlgorithmInteger(pp, "maxwell_fdtd_solver");
         field_gathering_algo = GetAlgorithmInteger(pp, "field_gathering");
-        if (field_gathering_algo == GatheringAlgo::MomentumConserving) {
+        if (field_gathering_algo == GatheringAlgo:M:omentumConserving) {
```

Comment:
Typo?

Reply:
Hm it looks like there was no typo in the original code, doesn't it?

Reply:
Oh sorry, yes that's true, should be fixed by now.